index.js

```diff
@@ -21,7 +103,9 @@ module.exports = (robot, _, Settings = require('./lib/settings')) => {
       return
     }
 
-    const config = await context.config('settings.yml', {}, { arrayMerge: mergeArrayByName })
-    return Settings.sync(context.github, context.repo(), config)
+    await onPush(context)
+    if (repositoryName === '.github') {
```

Comment:
Do you think we could make this configurable in case I don't want to name my .github repository ".github"? The main reason for that is that I have a [master repo](https://github.com/TokTok/toktok-stack) which has submodules with all the repos in the org. I currently named my "base" repo [template](https://github.com/TokTok/template), but I'm not married to that. However, I can't name it ".github", because then I can't have a submodule for it in the main stack repo, at least not have it named the same way as the .github repo itself. I could have a submodule "template" that points at the repo ".github", but it would be anomalous compared to everything else.

Reply:
these are the types of details that make this implementation complex. thank you for raising this so that we can consider it. i wasnt personally aware that probot-config supported extending config from other repos than `.github`, but if that is supported there, it probably makes sense to find a way to enable when this goes out. 

would you mind capturing those thoughts in #95 in case this PR ends up not being the way we move this effort forward?